src/state/therapies/reducer.ts

```diff
@@ -25,6 +25,12 @@ export default produce((draft: ITherapiesState, action: IAction<any, any>) => {
     case CREATE_THERAPY_SUCCESS: {
       draft.createTherapy.status = "SUCCESS";
       draft.createTherapy.data = action.payload;
+      if (draft.therapiesByPatientId.data)
+        draft.therapiesByPatientId.data = [
+          ...draft.therapiesByPatientId.data,
+          action.payload,
+        ];
+      else draft.therapiesByPatientId.data = [action.payload];
```

Comment:
Please, here add the curly brackets to the if / else statement. After that shloud be OK.